lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -44,8 +44,8 @@
         teamMembershipDetailUrl: '${ team_membership_detail_url }',
         myTeamsUrl: '${ my_teams_url }',
         maxTeamSize: ${ course.teams_max_size },
```

Comment:
This doesn't affect this code review specifically, but I wanted to call out this maxTeamSize example as something that will require a decision before someone can close up the future stories.

Proposal 1:
```
# Make string here, and parse to number where used
maxTeamSize: '${ escape_js_str(course.teams_max_size) | n }',
```
Pros:
- No additional methods
- In case data might be manipulated by a user, it will be protected
Cons:
- Additional parse to number may be needed
- Additional quotes needed around number, which may be misleading

Proposal 2:
```
# 2.1 Ensure number
maxTeamSize: ${ ensure_number(course.teams_max_size) | n },
or
# 2.2 Ensure number
maxTeamSize: ${ escape_js_number(course.teams_max_size) | n },
```
Pros:
- Adds type checking protection without changing any other code
Cons:
- Introduce another method and method naming question
- 2.1: I wanted consistency of "escape" for all params.
- 2.2: I don't like that "escape" isn't really escaping.

@andy-armstrong @cahrens @nedbat Ultimately any decision around this should be moved to the Confluence page, but I thought I'd bring up the topic here in context.